src/app/pages/home/home.component.html

```diff
@@ -1,2 +1,4 @@
-
-  <app-nav-bar></app-nav-bar>
+<section>
+ <a routerLink="/teams"><span>Ce lien</span></a>
```

Comment:
ce lien ? name it as the page you refer to like team or about us

Reply:
about us